core/modules/replica/DatabaseMySQL.cc

```diff
@@ -1,6 +1,6 @@
 /*
  * LSST Data Management System
- * Copyright 2017 LSST Corporation.
+ * Copyright 2018 LSST Corporation.
```

Comment:
I think this should say `2017-2018` ...?

Reply:
This probably won't matter. Remember a discussion we had with Tim on this subject when you was reviewing my previous (huge) ticket? He suggested we should move the Copyright into into a separate (one-per folder or GitHub package file). Someone just has to do this :-)

Reply:
Actually, I've just found that the 'replica' module already has file COPYRIGHT which I myself created following Tim's suggestion! So, I'm going to get rid of the incorrect Copyrights from all header files.